transmart-schemas/src/main/resources/db/changelog/changes/20191907081438_biomart_user_views_with_bitset.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<!--
+  ~ Copyright (c) 2019  The Hyve B.V.
+  ~  This file is distributed under the GNU General Public License
+  ~  (see accompanying file LICENSE).
+  -->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+    <preConditions onFail="WARN">
+        <dbms type="PostgreSQL" />
+        <sqlCheck expectedResult="1">
+            SELECT count(*) FROM pg_extension WHERE extname='pg_bitcount'
+        </sqlCheck>
+    </preConditions>
+    <changeSet author="artur@thehyve.nl" id="20191907081438-0" dbms="PostgreSQL">
+        <comment>
+            Materialized views are not supported yet by liquibase
+        </comment>
+        <sql endDelimiter=";">
+            CREATE MATERIALIZED VIEW biomart_user.study_concept_bitset TABLESPACE pg_default AS
```

Comment:
I think the `TABLESPACE pg_default` and `WITH DATA` parts are not needed.

Reply:
let's check

Reply:
looks like if it works. I remove these things, and get the same materialized view.